starter-code/src/clue.js

```diff
@@ -1,14 +1,186 @@
+		color: "purple"
+	},
+
+	{
+		name: "missScarlet",
+		firstName: "Kasandra",
+		lastName: "Scarlet",
+		occupation: "Actor",
+		age: 31,
+		description: "She is an A-list movie star with a dark past",
+		image: "https://www.radiotimes.com/uploads/images/Original/111967.jpg",
+		color: "red"
+	},
+	{
+		name: "mrsPeacock",
+		firstName: "Eleanor",
+		lastName: "Peacock",
+		occupation: "Socialité",
+		age: 36,
+		description: "She is from a wealthy family and uses her status and money to earn popularity",
+		image: "https://metrouk2.files.wordpress.com/2016/07/mrs-peacock.jpg",
+		color: "blue"
+	},
+
+	{
+		name: "mrMustard",
+		firstName: "Jack",
+		lastName: "Mustard",
+		occupation: "Retired Football player",
+		age: 62,
+		description: "He is a former football player who tries to get by on his former glory",
+		image: "https://static.independent.co.uk/s3fs-public/thumbnails/image/2016/07/04/08/unspecified-3.jpg",
+		color: "yellow"
+	}
+];
+
+console.table(suspectsArray);
+
+//Rooms Collection as object
+const roomsArray = [
+	{ name: "Dining Room" },
+	{ name: "Conservatory" },
+	{ name: "Kitchen" },
+	{ name: "Study" },
+	{ name: "Library" },
+	{ name: "Billiard Room" },
+	{ name: "Lounge" },
+	{ name: "Ballroom" },
+	{ name: "Hall" },
+	{ name: "Spa" },
+	{ name: "Living Room" },
+	{ name: "Observatory" },
+	{ name: "Theater" },
+	{ name: "Guest House" },
+	{ name: "Patio" }
+];
 
 // Rooms Collection
-const roomsArray = [];
+// const roomsArray = [
+// 	"Dining Room",
+// 	"Conservatory",
+// 	"Kitchen",
+// 	"Study",
+// 	"Library",
+// 	"Billiard Room",
+// 	"Lounge",
+// 	"Ballroom",
+// 	"Hall",
+// 	"Spa",
+// 	"Living Room",
+// 	"Observatory",
+// 	"Theater",
+// 	"Guest House",
+// 	"Patio"
+// ];
 
 // Weapons Collection
-const weaponsArray = [];
+const weaponsArray = [
+	{
+		name: "rope",
+		weight: 10
+	},
+	{
+		name: "knife",
+		weight: 8
+	},
+	{
+		name: "candlestick",
+		weight: 2
+	},
+	{
+		name: "dumbbell",
+		weight: 30
+	},
+	{
+		name: "poison",
+		weight: 2
+	},
+	{
+		name: "axe",
+		weight: 15
+	},
+	{
+		name: "bat",
+		weight: 13
+	},
+	{
+		name: "trophy",
+		weight: 25
+	},
+	{
+		name: "pistol",
+		weight: 20
+	}
+];
 
 // ITERATION 2
 
+function selectRandom(suspectsArray) {
+	let suspect = suspectsArray[Math.floor(Math.random() * suspectsArray.length)];
+
+	return suspect;
+}
+let suspect = "";
+
+function pickMystery() {
+	let weapon = selectRandom(weaponsArray);
+	let room = selectRandom(roomsArray);
+
+	let suspect = selectRandom(suspectsArray);
+
+	let mystery = { weapon: weapon, suspect: suspect, room: room };
+	return mystery;
+}
 // ITERATION 3
+
+function revealMystery(mystery) {
+	let last = mystery.suspect.lastName;
+	let first = mystery.suspect.firstName;
+	let weapon = mystery.weapon.name;
+	let location = mystery.room.name;
+
+	x = first + " " + last + " killed Mr. Boddy using the " + weapon + " in the " + location + "!";
```

Comment:
use backticks and $ sign. Will make your code cleaner :) Also think about how you name your variables. Maybe e.g.`output` would be better than `x`. Also make sure to declare it using var, const or let.

```
const output = `${first} ${last} killed Mr. Boddy using the ${weapon} in the ${location}!`
```